### PR TITLE
Updated most of ar.json

### PR DESCRIPTION
**Pull request by EsmailELBoBDev2**

### CODE REVIEW

File: locales/ar.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
     "newest": "الأجدد",
     "oldest": "الأقدم",
     "popular": "الاكثر شعبية",
-    "last": "اخر الفيديوهات المعدلة",
+    "last": "اخر قوائم التشغيل المعدلة (",
```
Comment (omarroth) [inline locales/ar.json]: Is the trailing `(` a typo or part of the translation?

**Comment by omarroth** [comment]
Just a question on one of the translations, otherwise LGTM :+1:

**Comment by EsmailELBoBDev2** [comment]
well idk what i did but i guess i removed ur question xD & no i just forgot to remove it

**Comment by omarroth** [comment]
:+1:

**Pull request merged**